src/stampy_nlp/faq_titles.py

```diff
@@ -82,11 +90,16 @@ def encode_faq_titles():
 
 
 if __name__ == "__main__":
+    from stampy_nlp.models import connect_pinecone
+    from stampy_nlp.utilities.pinecone_utils import get_index
+
     logging.basicConfig(level=logging.DEBUG)
```

Comment:
Even with the level set at DEBUG, I'm only seeing INFO level messages, not the full debug. What else do I need to do?

Reply:
it gets overwritten by env variables. Check in `.env` whether you have `LOG_LEVEL` set, or prepend the execution with `LOG_LEVEL=DEBUG`

Reply:
Not set in .env but I added it and works great now!